### PR TITLE
Build pre-built distribution + update dependencies

### PR DESCRIPTION
**Pull request by barmac**
This pull request adds pre-built distribution and re-adds `package-lock.json` following the `bpmn-moddle`'s example and also updates dependencies.

### CODE REVIEW

**Comment by barmac** [comment]
I've already confirmed that it works nicely with `dmn-js@develop`.

File: rollup.config.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    json(),
+    ...plugins
+  ];
+}
+
+const srcEntry = pkg.source;
+
+const umdDist = pkg['umd:main'];
+
+const umdName = 'BpmnModdle';
```
Comment (nikku) [inline rollup.config.js]: Did not give it a full review. This one looks broken though.

**Comment by nikku** [comment]
Goal (to be validated by the reviewer): Should work exactly as `bpmn-moddle`.

File: rollup.config.js (hunk 1 above)
Comment (barmac) [reply rollup.config.js]: Sure, I will correct this!

**Pull request merged**